apps/consoledot-rhosak/src/routes/control-plane/routes/KafkaInstancesRoute.tsx

```diff
@@ -40,6 +43,20 @@ export const KafkaInstancesRoute: FunctionComponent<KafkaInstancesRoute> = ({
 
   const labels = useKafkaLabels();
 
+  const { auth } = useChrome();
+
+  const [currentUser, setCurrentUser] = useState<string>();
+  const [isOrgAdmin, setIsOrgAdmin] = useState<boolean>(false);
+
+  useEffect(() => {
+    const getUser = async () => {
+      const user = await auth.getUser();
+      setCurrentUser(user?.identity?.user?.username ?? "");
+      setIsOrgAdmin(user?.identity?.user?.is_org_admin ?? false);
+    };
+    void getUser();
+  }, [auth]);
```

Comment:
I'd make this a fetcher in the api, with its own useUser hook. So it's cached and it's ready to be reused.

Just remember to add the new dependency on the api package.